spec/domain/command_handlers/game_command_handler_spec.rb

```diff
@@ -116,231 +137,335 @@
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.u_turn(40)),
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.u_turn(50)),
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.u_turn(60)),
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(70)),
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(90)),
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(110)),
+          RobotProgrammedEvent.new(id, bob, [
+            InstructionCard.u_turn(10),
+            InstructionCard.u_turn(20),
+            InstructionCard.u_turn(30),
+            InstructionCard.u_turn(40),
+            InstructionCard.u_turn(50)
+          ]),
+          AllRobotsProgrammedEvent.new(id)
+        )
+      end
 
-        let(:expected_events) do
-          [
-            RobotRotatedEvent.new(id, bob, GameUnit.new(2, 1, GameUnit::UP)),
-            RobotRotatedEvent.new(id, bob, GameUnit.new(2, 1, GameUnit::DOWN)),
-            RobotRotatedEvent.new(id, bob, GameUnit.new(2, 1, GameUnit::UP)),
-            RobotRotatedEvent.new(id, bob, GameUnit.new(2, 1, GameUnit::DOWN)),
-            RobotRotatedEvent.new(id, bob, GameUnit.new(2, 1, GameUnit::UP)),
-            InstructionCardDiscardedEvent.new(id, InstructionCard.u_turn(10)),
-            InstructionCardDiscardedEvent.new(id, InstructionCard.u_turn(20)),
-            InstructionCardDiscardedEvent.new(id, InstructionCard.u_turn(30)),
-            InstructionCardDiscardedEvent.new(id, InstructionCard.u_turn(40)),
-            InstructionCardDiscardedEvent.new(id, InstructionCard.u_turn(50)),
-            InstructionCardDiscardedEvent.new(id, InstructionCard.u_turn(60)),
-            InstructionCardDiscardedEvent.new(id, InstructionCard.rotate_left(70)),
-            InstructionCardDiscardedEvent.new(id, InstructionCard.rotate_left(90)),
-            InstructionCardDiscardedEvent.new(id, InstructionCard.rotate_left(110)),
-            GameRoundStartedEvent.new(
-              id, GameRound.new(2), {'bob' => []}, {'bob' => []}
-            ),
-            InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(130)),
-            InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(150)),
-            InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(170)),
-            InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(190)),
-            InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(210)),
-            InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(230)),
-            InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(250)),
-            InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(270)),
-            InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(290))
-          ]
-        end
+      it 'plays the current round' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.